README.md

```diff
@@ -1,7 +1,7 @@
 # mattermost-plugin-api
 
-[![GoDoc](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)](https://pkg.go.dev/github.com/lieut-data/mattermost-plugin-api)
-[![Test Status](https://github.com/lieut-data/mattermost-plugin-api/workflows/Test/badge.svg)](https://github.com/lieut-data/mattermost-plugin-api/actions?query=workflow%3ATest)
+[![GoDoc](https://img.shields.io/static/v1?label=godoc&message=reference&color=blue)](https://pkg.go.dev/github.com/mattermost/mattermost-plugin-api)
```

Comment:
https://pkg.go.dev/github.com/mattermost/mattermost-plugin-api should resolve soon